packages/graphback/src/index.ts

```diff
@@ -1,4 +1,5 @@
 export * from '@graphback/core'
 export * from '@graphback/runtime'
+export * from './GraphbackConfig'
 export * from './GraphbackGenerator'
 export * from './GraphbackRuntime'
```

Comment:
I think eventually like to hide this class from the application consuming it and have a helper function do the class creation. You will probably see from the runtime templates how much work is involved in configuring Graphback runtime from the template but most of this can happen from our package.

https://github.com/aerogear/graphback/issues/1071

Reply:
Yeah I agree. The something like a generic version of the `createRuntime` function in the actual runtime template itself. To be honest, the whole thing could be a function the whole way down but I won't start a flame war 😅